Gems/ScriptCanvas/Code/Editor/SystemComponent.h

```diff
@@ -14,15 +14,18 @@
 #include <AzCore/Jobs/JobManager.h>
 #include <AzCore/UserSettings/UserSettingsProvider.h>
 #include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <AzToolsFramework/API/EditorAssetSystemAPI.h>
 #include <AzToolsFramework/API/ToolsApplicationAPI.h>
 #include <AzToolsFramework/Asset/AssetSeedManager.h>
 #include <AzToolsFramework/AssetBrowser/AssetBrowserBus.h>
 #include <AzToolsFramework/Editor/EditorContextMenuBus.h>
+#include <AzToolsFramework/Entity/EditorEntityContextBus.h>
+#include <Builder/ScriptCanvasBuilder.h>
 #include <Core/GraphBus.h>
 #include <Editor/View/Windows/Tools/UpgradeTool/Model.h>
 #include <ScriptCanvas/Bus/ScriptCanvasBus.h>
 #include <ScriptCanvas/Bus/ScriptCanvasExecutionBus.h>
-#include <AzToolsFramework/Entity/EditorEntityContextBus.h>
+#include <Builder/ScriptCanvasBuilderDataSystem.h>
```

Comment:
nit: can you sort these?

Reply:
that is never a nit. After the current PRs close, I'm doing a full sort pass on a couple of files that are really disorganized.